docs/contracts/inheritance.rst

```diff
@@ -172,6 +176,48 @@ virtual method lookup.
 
 .. _constructor:
 
+Function Overriding
+===================
+
+Base functions can be overridden by inheriting contracts to change their
+behavior. The overriding function must then use the ``override`` keyword in the
```

Comment:
What is the currently implemented logic with regards to 3-layer inheritance where the middle contract does not mention the function?

Reply:
Then please explain it here.

Reply:
Do you consider that behavior uncommon? I figured that's the expected behavior and only if it behaves differently it would be worth documenting..

Reply:
Below, you use the word "direct base", so I assume when you use "base" here, you mean "direct or indirect base". This makes me conclude that it is allowed to leave out function definitions in the middle contract. If that is so, I would still like this to be stated explicitly because it will reduce a big amount of confusion.

Reply:
my intention was that one should use the most derived base. I'll try to rephrase it again and I'll add this point, too